x-pack/test_utils/testbed/testbed.js

```diff
@@ -4,20 +4,75 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { Component } from 'react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import PropTypes from 'prop-types';
 import { Provider } from 'react-redux';
 import { mountWithIntl } from '../enzyme_helpers';
-import { findTestSubject as findTestSubjectHelper } from '@elastic/eui/lib/test';
+import { findTestSubject as findTestSubjectHelper } from '../index';
 
 const registerTestSubjExists = component => (testSubject, count = 1) => findTestSubjectHelper(component, testSubject).length === count;
 
-export const registerTestBed = (Component, defaultProps, store = {}) => (props) => {
+const defaultOptions = {
+  memoryRouter: {
+    wrapRoute: true,
+  },
+};
+
+const withRoute = (WrappedComponent, componentRoutePath = '/', onRouter = () => {}) => {
+  return class extends Component {
+      static contextTypes = {
```

Comment:
Minor nit: this looks a little over-indented.

Reply:
Eagle eyes too!